benchmark/CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 file(GLOB BENCHMARK "*.cpp")
 add_executable( benchmark ${BENCHMARK} )
 
-target_link_libraries( benchmark fc Boost::program_options bn256)
+target_link_libraries( benchmark eosio_testing fc Boost::program_options bn256)
 target_include_directories( benchmark PUBLIC
                             "${CMAKE_CURRENT_SOURCE_DIR}"
+                            "${CMAKE_CURRENT_BINARY_DIR}/../unittests/include"
```

Comment:
We don't need to fix this now (or even when merging to main) but we should try to avoid these sort of declarations. We probably want a target similar to `eosio_testing_contracts`,
https://github.com/AntelopeIO/leap/blob/581cd2b605ea0f8c2e21493a2647d02e31047dfc/libraries/testing/CMakeLists.txt#L39-L41

Reply:
Thanks @spoonincode. I was doing the cleos set code issue yesterday and not getting around with this. As the PR has already merged into 5.0, for cleaner tracking, I will merge the PR to main as is, and use your suggested way later.